content/developer/composer/getting-started.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: bt_wiki
+title: Getting started
+category: Cloudify Composer
+draft: false
+weight: 300
+aliases: ["/composer/installation/", "/developer/composer/installation/"]
+---
+
+
+Starting from Cloudify 4.1, the {{< param cfy_composer_name >}} comes as part of the {{< param mgr_premium_title >}}.
+
+
+## Starting
+
+You can access {{< param cfy_composer_name >}} from your browser:
+ 
+ * directly via `http://<cloudify-manager-ip>/composer` or `https://<cloudify-manager-ip>/composer` 
+ * from {{< param cfy_console_name >}} by clicking on the **{{< param cfy_composer_name >}}** link button at Local Blueprints page
+
```

Comment:
Maybe it's worth to mention "Edit a copy in composer" feature here?

Reply:
Right, done.